src/nti/transactions/transactions.py

```diff
@@ -237,6 +253,16 @@ def do(*args, **kwargs):
     transaction.get().join(result)
     return result
 
+def do_last(*args, **kwargs):
```

Comment:
Can we not use the word "last", here or in the class? That implies promises we can't keep, and it makes no sense if this function/class is used more than once.